pdns/misc.cc

```diff
@@ -198,6 +198,30 @@ size_t writen2WithTimeout(int fd, const void * buffer, size_t len, const struct
   return len;
 }
 
+auto pdns::getMessageFromErrno(const int errnum) -> std::string
```

Comment:
Perhaps ponder using this function in `stringerror`?

Reply:
I would leave this sort of change up to a cleanup round at some point if that's OK?
We'd replace the use of `stringerror` with `getMessageFromErrno`.

Reply:
Sure, a separate PR for this at a later time works for me!